tests/support/identity.rs

```diff
@@ -29,44 +30,88 @@ type Certify = Box<
         + Send,
 >;
 
-pub fn rsp_from_cert<P>(p: P) -> Result<pb::CertifyResponse, io::Error>
-where
-    P: AsRef<Path>,
-{
-    let f = fs::File::open(p)?;
-    let mut r = io::BufReader::new(f);
-    let certs = rustls::internal::pemfile::certs(&mut r)
-        .map_err(|_| io::Error::new(io::ErrorKind::Other, "rustls error reading certs"))?;
-    let leaf_certificate = certs
-        .get(0)
-        .ok_or_else(|| io::Error::new(io::ErrorKind::Other, "no certs in pemfile"))?
-        .as_ref()
-        .into();
-    let intermediate_certificates = certs[1..].iter().map(|i| i.as_ref().into()).collect();
-    let rsp = pb::CertifyResponse {
-        leaf_certificate,
-        intermediate_certificates,
-        ..Default::default()
-    };
-    Ok(rsp)
+const TLS_VERSIONS: &[rustls::ProtocolVersion] = &[rustls::ProtocolVersion::TLSv1_2];
+
+struct Certificates {
+    pub leaf_certificate: Vec<u8>,
+    pub intermediate_certificates: Vec<Vec<u8>>,
```

Comment:
Since the struct is `Certificates`, maybe we call the fields `leaf` and `intermediates`?